yt_dlp/extractor/twitcasting.py

```diff
@@ -142,7 +142,7 @@ def _real_extract(self, url):
             'https://twitcasting.tv/streamserver.php?target=%s&mode=client' % uploader_id, video_id,
             'Downloading live info', fatal=False)
 
-        is_live = 'data-status="online"' in webpage
+        is_live = 'data-status="recorded"' in webpage
```

Comment:
for backwards compat in case the site's change is reverted
```suggestion
        is_live = 'data-status="recorded"' in webpage or 'data-status="online"' in webpage
```

Reply:
```suggestion
        is_live = any(f'data-{x}' in webpage for x in ['is-onlive="true"', 'live-type="live"', 'status="online"'])
```